src/map/unit.cpp

```diff
@@ -1858,10 +1858,7 @@ int unit_skilluse_id2(struct block_list *src, int target_id, uint16 skill_id, ui
 		}
 	}
 
-	if (src->type == BL_NPC) // NPC-objects can override cast distance
-		range = AREA_SIZE; // Maximum visible distance before NPC goes out of sight
-	else
-		range = skill_get_range2(src, skill_id, skill_lv, true); // Skill cast distance from database
+	range = skill_get_range2(src, skill_id, skill_lv, true); // Skill cast distance from database
```

Comment:
Sorry, when I told you to revert this I should have also mentioned throwing this inside that if-statement you adjusted as well, since that's the only place that uses range.

So if you could move the whole `int range = <blah>` below that would be great!